### ISSUE TITLE
Update GeneticCondition model of present vs absent phenotypes

### ISSUE THREAD

**Issue by mbrush** [clingen-data-model/clingen-interpretation#216]
As we are now reserving the negation/NOT qualifier pattern to apply to statement outcomes, and the phenotypes linked to a GeneticCondition are not statement outcomes, consider how we might represent phenotype asserted to be absent from a particular condition.

One approach would be to use separate attributes to describe phenotypes observed and not observed in the condition, e.g. ' GeneticCondition.presentPhenotype' and 'GeneticCondition.absentPhenotype' (instead of a single GeneticCondition.phenotypes attribute that would have to be negated with a NOT qualifier if the indicated phenotype as absent)

**Comment by rrfreimuth** [comment]
That sounds like a reasonable approach.  I presume absentPhenotype means the phenotype was assessed and not found, as opposed to simply not looking for it.  Do we need to support the latter as well?


On Apr 25, 2018, at 7:23 PM, Matthew Brush <notifications@github.com<mailto:notifications@github.com>> wrote:


As we are now reserving the negation/NOT qualifier pattern to apply to statement outcomes, and the phenotypes linked to a GeneticCondition are not statement outcomes, consider how we might represent phenotype asserted to be absent from a particular condition.

One approach would be to use separate attributes to describe phenotypes observed and not observed in the condition, e.g. ' GeneticCondition.presentPhenotype' and 'GeneticCondition.absentPhenotype' (instead of a single GeneticCondition.phenotypes attribute that would have to be negated with a NOT qualifier if the indicated phenotype as absent)

**Comment by mbrush** [comment]
Agreed to move ahead with this proposal on 4-26-18 Call.  Will split the GeneticCondition.phenotype attribute into two attributes:

1. GeneticCondition.presentPhenotype (test for and observed) 
2. GeneticCondition.absentPhenotype (tested for and not observed)

**Comment by larrybabb** [comment]
@mbrush I'm pushing back on this because I have not seen this decision make its way into our final "end game" worksheet. 
Let's quickly review where we left off with this.
